style.css

```diff
@@ -21,6 +21,19 @@ body {
     padding: 0px;
 }
 
+button {
```

Comment:
We should always try to make our styles reusable (and composable).

When we develop websites or apps, there are common UI elements that are used throughout, like buttons, cards, text fields, etc.

So when we style our markup, we should think about what these things look like as our baseline. Then, we can create more specific styles for cases where these elements are used in interesting ways.